resources/views/cart/overview.blade.php

```diff
@@ -5,93 +5,112 @@
+                                        height="100"
+                                    />
+                                </a>
+                            </div>
+                            <div class="w-5/6 sm:w-5/12 xl:w-6/12">
+                                <a
+                                    class="font-bold"
+                                    :href="item.url | url"
+                                    dusk="cart-item-name"
+                                >@{{ item.name }}</a>
+                                <div v-for="(optionValue, option) in item.options">
+                                    @{{ option }}: @{{ optionValue }}
+                                </div>
+                                <div v-for="option in cart.items2.find((item) => item.item_id == itemId).options.filter((option) => !['info_buyRequest', 'option_ids'].includes(option.code))">
+                                    @{{ option.label }}: @{{ option.value.title || option.value }}
+                                </div>
+                            </div>
+                            <div class="w-2/6 pr-5 text-right sm:w-1/6 xl:w-2/12">
+                                @{{ item.price | price }}
+                            </div>
+                            <div class="w-2/6 sm:w-1/6 xl:w-1/12">
+                                <div class="inline-flex lg:w-3/4 xl:w-full">
+                                    <x-rapidez::input
+                                        class="text-right"
+                                        name="qty"
+                                        type="number"
+                                        :label="false"
+                                        v-model="item.qty"
+                                        v-bind:dusk="'qty-'+index"
+                                        ::min="item.min_sale_qty > item.qty_increments ? item.min_sale_qty : item.qty_increments"
+                                        ::step="item.qty_increments"
+                                    />
+                                    <x-rapidez::button
+                                        class="ml-1"
+                                        :title="__('Update')"
+                                        v-on:click="changeQty(item)"
+                                        v-bind:dusk="'item-update-'+index"
+                                    >
+                                        <x-heroicon-s-arrow-path class="h-4 w-4" />
+                                    </x-rapidez::button>
+                                </div>
+                            </div>
+                            <div class="flex w-2/6 items-center justify-end text-right sm:w-1/6 xl:w-2/12">
+                                @{{ item.total | price }}
+                                <a
+                                    class="ml-2"
+                                    href="#"
```

Comment:
since you remove the item with JS why make it a <a with an href="#, is a <button not better for this? Also you can remove the .prevent after the click if i'm correct